### ISSUE TITLE
Invalid parameter: 'key_source' Apt::Source[nginx] at /etc/puppetlabs/code/modules/nginx/manifests/package/debian.pp:37

### ISSUE THREAD

**Issue by brandfocus** [jfryman/puppet-nginx#629]

**Comment by 3flex** [comment]
Fixed in https://github.com/jfryman/puppet-nginx/pull/612

@jfryman consider pushing a new release to Forge?

**Comment by 3flex** [comment]
In the meantime @brandfocus you can downgrade puppetlabs-apt to 1.8.0

**Comment by bhechinger** [comment]
Also you may want to double check puppetlabs/concat. The current version is higher than what this module allows.

**Comment by 3flex** [comment]
Thanks @bhechinger. The new version of concat had a couple of issues, see #624, but that seems to have been resolved with the exception of a couple of deprecation notices that can be resolved in a future PR.

I've just updated that dependency in the metadata file.

@jfryman any chance of a new release? To be Puppet Approved Forge and source can't be more than a month apart, and the last release was April 7.
